### DISCUSSION TITLE
Avoir des règles plus spécifiques

### DISCUSSION THREAD

**Discussion by EmmanuelDemey** [cnumr/best-practices#414] [General]
En se basant sur les règles actuelles, est il prévu d'étoffer ce repository avec des règles plus spécifiques ? C'est en effet un travail que nous sommes en train de faire avec l'un de mes clients. 

Le but serait savoir des règles plus spécifiques que les lecteurs (juste de ce repo) pour prendre en main et utiliser pour auditer plus facilement leurs service numérique. 

Un exemple : limiter le poid de la page, on pourrait avoir une règle spécifique sur l'utilisation de Moment, ou la mise en place du tree shaking, du lazy loding de page, mauvaise configuration de lodash, ...

**Comment by EmmanuelDemey** [comment]
comment pouvons nous formuler cela dans la nomenclature actuelle ?

**Reply by rlemaire** [reply]
Hello Emmanuel,
Il n'y a rien de prévu sur ce sujet à ce stade.
Mais tu peux utiliser le mécanisme normal d'issues et de PR pour proposer des choses.

**Reply by EmmanuelDemey** [reply]
Je viens de faire une PR avec une première règle https://github.com/cnumr/best-practices/pull/437
